pkg/action/executor/move_util.go

```diff
@@ -478,20 +484,23 @@ func resourceRollout(client dynamic.ResourceInterface, obj *unstructured.Unstruc
 	return nil
 }
 
-// changeScheduler sets the scheduler value to default or unsets it to an invalid one
-func changeScheduler(client dynamic.ResourceInterface, obj *unstructured.Unstructured, valid bool) error {
+// ChangeScheduler sets the scheduler value to default or unsets it to an invalid one
+// This additionally adds or removes the garbage collection label on the resource
+func ChangeScheduler(client dynamic.ResourceInterface, obj *unstructured.Unstructured, valid bool) error {
```

Comment:
Can we also split this method to two methods, i.e., ChangeScheduler and RevertScheduler ? It's clearer to me compared to using valid / invalid (nothing is invalid here)

Reply:
Please see the comment below (for which I did create an example). The same applies here.
Would a better naming (invalid -> incorrect/bad/somethingelse) explain it more? 
Meanwhile the intention is indeed set a name of a scheduler to something `invalid` or `bad`. That means if the cluster for some advanced reasons also has a scheduler named `turbo-scheduler`, we will need to use some other `invalid` name in place. `ChangeScheduler` and `RevertScheduler` does not make this explicit, rather might make it sound that we are using another scheduler to the reader of the code without context.